cli/cmd/report_rules.go

```diff
@@ -0,0 +1,409 @@
+			}
+			if len(reportRules.Data) == 0 {
+				msg := `There are no report rules configured in your account.
+
+Get started by configuring your report rules using the command:
+
+    lacework report-rule create
+
+If you prefer to configure report rules via the WebUI, log in to your account at:
+
+    https://%s.lacework.net
+
+Then navigate to Settings > Report Rules.
+`
+				cli.OutputHuman(fmt.Sprintf(msg, cli.Account))
+				return nil
+			}
+			if cli.JSONOutput() {
+				return cli.OutputJSON(reportRules)
+			}
+
+			var rows [][]string
+			for _, rule := range reportRules.Data {
+				rows = append(rows, []string{rule.Guid, rule.Filter.Name, rule.Filter.Status()})
+			}
+
+			cli.OutputHuman(renderSimpleTable([]string{"GUID", "NAME", "ENABLED"}, rows))
+			return nil
+		},
+	}
+	// show command is used to retrieve a lacework report rule by guid
+	reportRulesShowCommand = &cobra.Command{
+		Use:   "show",
+		Short: "Show a report rule by id",
+		Long:  "Show a single report rule by it's id.",
+		Args:  cobra.ExactArgs(1),
+		RunE: func(_ *cobra.Command, args []string) error {
+			var response api.ReportRuleResponse
+			cli.StartProgress(" Fetching report rule...")
+			defer cli.StopProgress()
+
+			err := cli.LwApi.V2.ReportRules.Get(args[0], &response)
+			if err != nil {
+				return errors.Wrap(err, "unable to get report rule")
+			}
+
+			if cli.JSONOutput() {
+				return cli.OutputJSON(response)
+			}
+
+			reportRule := response.Data
+			headers := [][]string{
+				[]string{reportRule.Guid, reportRule.Filter.Name, reportRule.Filter.Status()},
+			}
+
+			cli.OutputHuman(renderSimpleTable([]string{"GUID", "NAME", "ENABLED"}, headers))
+			cli.OutputHuman("\n")
+			cli.OutputHuman(buildReportRuleDetailsTable(reportRule))
+
+			return nil
+		},
+	}
+
+	// delete command is used to remove a lacework report rule by id
+	reportRulesDeleteCommand = &cobra.Command{
+		Use:   "delete",
+		Short: "Delete a report rule",
+		Long:  "Delete a single report rule by it's ID.",
+		Args:  cobra.ExactArgs(1),
+		RunE: func(_ *cobra.Command, args []string) error {
+			cli.StartProgress(" Deleting report rule...")
+			defer cli.StopProgress()
+
+			err := cli.LwApi.V2.ReportRules.Delete(args[0])
+			if err != nil {
+				return errors.Wrap(err, "unable to delete report rule")
+			}
+			cli.OutputHuman(fmt.Sprintf("The report rule with GUID %s was deleted\n", args[0]))
+			return nil
+		},
+	}
+
+	// create command is used to create a new lacework report rule
+	reportRulesCreateCommand = &cobra.Command{
+		Use:   "create",
+		Short: "Create a new report rule",
+		Args:  cobra.NoArgs,
+		RunE: func(_ *cobra.Command, args []string) error {
+			if !cli.InteractiveMode() {
+				return errors.New("interactive mode is disabled")
+			}
+
+			response, err := promptCreateReportRule()
+			if err != nil {
+				return errors.Wrap(err, "unable to create report rule")
+			}
+
+			cli.OutputHuman(fmt.Sprintf("The report rule was created with GUID %s\n", response.Data.Guid))
```

Comment:
You don't need to format the string, the underlying function does it automatically.
```suggestion
			cli.OutputHuman("The report rule was created with GUID %s\n", response.Data.Guid)
```